snapshot-labs/sx-monorepo
# feat: add mobile menu

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/212

### How to test

1. Open website
2. On md+ screen, same behavior as before
3. On smaller screen, there should be a new bottom menu when sidebar is open

### TODO

- [x] Depends on https://github.com/snapshot-labs/sx-monorepo/pull/792
